### DISCUSSION TITLE
How to set the size of a QR code generated by the bb-qr-code plugin

### DISCUSSION THREAD

**Discussion by ecxs** [Budibase/budibase#8768] [Help]
Hi there,

Does anybody know how to set the size of a QR code that is generated with the help of the bb-qr-code plugin?

Thanks in advance!

**Comment by melohagan** [comment]
Hey @ecxs 

It may be best to raise an issue on their repository!

**Reply by ecxs** [reply]
@melohagan just did, and will post the results ...
